bind_test.go

```
@@ -988,6 +989,48 @@
+		require.Len(t, cq.Data, 2)
+		require.Equal(t, "john", cq.Data[0].Name)
+		require.Equal(t, "doe", cq.Data[1].Name)
+	})
+
+	t.Run("MultipartCollectionQuerySquareBrackets", func(t *testing.T) {
+		c := app.AcquireCtx(&fasthttp.RequestCtx{})
+		c.Request().Reset()
+
+		buf := &bytes.Buffer{}
+		writer := multipart.NewWriter(buf)
+		writer.WriteField("data[0][name]", "john")
```

golangci-lint (1.62.2)

Error return value of `writer.WriteField` is not checked

(errcheck)